plugins/GSdx/Renderers/Vulkan/vulkanloader.cpp

```diff
@@ -0,0 +1,161 @@
+#include "vulkanloader.hpp"
+#include <stdio.h>
+#include <dlfcn.h>
+
+// Unfortunately what follows is a bit of a mess
+// so I'll explain it as we go for other people
+// who need to work in this file
+
+#ifdef VK_USE_PLATFORM_WIN32_KHR
+#define LoadFuncAddress GetProcAddress
+#else
+#define LoadFuncAddress dlsym
+#endif
+
+// This generates a list of variable defintions
+// for example
+// VK_GLOBAL_FUNC( vkGetInstanceProcAddr )
+// will expand to
+// PFN_vkGetInstanceProcAddr vkGetInstanceProcAddr;
+#define VK_GLOBAL_FUNC(func) PFN_##func func;
+#define VK_MODULE_FUNC(func) PFN_##func func;
+#define VK_INSTANCE_FUNC(func) PFN_##func func;
+#define VK_DEVICE_FUNC(func) PFN_##func func;
+#include "vulkan.inl"
+#undef VK_GLOBAL_FUNC
+#undef VK_MODULE_FUNC
+#undef VK_INSTANCE_FUNC
+#undef VK_DEVICE_FUNC
+
+// Differences in OS for handling
+// references to dynamically loaded
+// libraries
+
+// TODO: We will need to do this for
+// other libraries so it would be nice
+// to abstract this code elsewhere
+#ifdef VK_USE_PLATFORM_WIN32_KHR
+static HMODULE vulkan_module = nullptr;
+#else
+static void* vulkan_module = nullptr;
+#endif
+
+namespace Vulkan
+{
+    // Loads Vulkan
+    // as well as it's global and module functions
+    // Do to the nature of Vulkan we have to
+    // define different cats of function pointers
+    // depending on how we get them.
+    bool load_vulkan()
```

Comment:
Change call function names to CapitalCase

ie this will become `LoadVulkan()`

Reply:
Also GSdx coding style is tabs not spaces so switch that. Donno for visual studio code but in visual studio:
Select all
Edit -> Advanced -> Tabify Selected Lines